clang/lib/CIR/CodeGen/CIRGenFunction.h

```diff
@@ -110,6 +110,10 @@ class CIRGenFunction : public CIRGenTypeCache {
 public:
   mlir::Value createDummyValue(mlir::Location loc, clang::QualType qt);
 
+  void emitAggExpr(const clang::Expr *e, AggValueSlot slot);
```

Comment:
These should be moved to the place where other `emit*` functions are declared and inserted in alphabetical order.

Reply:
Done, thanks